FutureWorkshops/Smack
# Chore: Date Formatters, EN hardcoding

### :cop: Reviewers

@StelianMorariu  @thebestpol 

### :pushpin: References
N/A

### :tophat: What is the goal?

To make sure SMACK date formatting always uses the English Locale. As other Locales break queries that include date/time.


### How is it being implemented?

- Changed the date formatter utility in `JXMPP` https://github.com/FutureWorkshops/jxmpp/pull/1
- Replaced `JXMPP` dependencies with local modules, imported after the change above.


### How can it be tested?


This is a library, and you need to send a Date-based query to the server and verify it replies correctly.
